srcpkgs/woeusb/template

```diff
@@ -0,0 +1,46 @@
+# Template file for 'woeusb'
+pkgname=woeusb
+version=3.3.1
+revision=2
+wrksrc="WoeUSB-$version"
+archs=noarch
+build_style=gnu-configure
+configure_args="--with-wx-config=wx-config-gtk3"
+hostmakedepends="automake gettext libtool"
+makedepends="wxWidgets-gtk3-devel"
+depends="bash grub ntfs-3g parted wget"
+short_desc="Create a Windows USB stick installer from a real Windows DVD or image"
+maintainer="bra1nwave <bra1nwave@protonmail.com>"
```

Comment:
Also if you want you can take maintership for this package

Reply:
Nope, I happened to use this package for burning a Windows USB stick for boot.
Not really interested to follow their development.